components/raftstore-v2/src/operation/command/admin/split.rs

```diff
@@ -75,6 +75,9 @@ pub struct SplitResult {
     // The index of the derived region in `regions`
     pub derived_index: usize,
     pub tablet_index: u64,
+    // new regions will amortize the region size if it's true.
+    // otherwise, the new region's size will be 0.
+    pub amortize_source_region_size: bool,
```

Comment:
`amortize` means gradually write off the initial cost of (an asset) over a period. It's not proper here.

Reply:
+1

Reply:
I think the name can be size_based_split: bool

Reply:
has renamed `share_source_region_size `.